README.md

```diff
@@ -1,29 +1,131 @@
-# Web3 Unleashed 🐰 🐶 🐱 🐭 🐹
+# Unleash a NFT 🐰 🐶 🐱 🐭 🐹
 
-A boilerplate for the petaverse!
+In this tutorial, we will be creating a Bored Pets NFT smart contract for our Petaverse! The technologies we will be using are:
+
+- Truffle
+- Infura
+- Metamask
+- Web3.js
 
 ## Setup
 
+First, we'll want to build our Petaverse by cloning the unleashed repository's [main branch](https://github.com/trufflesuite/unleashed/tree/main). We recommend cloning it into a folder called `bored-pets-nft`. Follow the setup instructions [here]()
+
+You should have:
+1. The `unleashed` repository cloned
+2. Truffle installed
+3. Ganache installed
+4. An Infura account
+5. A MetaMask wallet
+
+<TODO: Put in set up instructions on base repo>
+
+Once you've gotten these installation requirements down, we can start building the NFT smart contract!
+
+## Core Concepts
+
+- A **NFT** (non-fungible token) is digital data that is stored on chain. Unlike cryptocurrencies, which are *fungible* (aka your Bitcoin is the same as my Bitcoin), NFTs are one-of-a-kind.
+- **ERC-721** defines the set of standards that make a token non-fungible. You can read more about it on Ethereum's website [here](https://ethereum.org/en/developers/docs/standards/tokens/erc-721/).
+- Every NFT will have a combination of a **`uint256 tokenId`** variable and contract address that must be globally unique.
+- All the fun stuff about NFTs - the rarity, description, image, etc - is part of its **metadata**. In this tutorial, the NFT's **`string tokenURI`** will point to where the metadata is stored on [IPFS](https://ipfs.io/).
+
+## Create the NFT smart contract
+
+Creating a NFT smart contract is short and sweet! [OpenZeppelin](https://www.openzeppelin.com/contracts) contracts make it super easy. First, we'll want change into our `packages/truffle` directory and add the OpenZeppelin dependency:
+
 ```bash
-$ yarn install
+cd packages/truffle
+yarn add @openzeppelin/contracts
 ```
+Then, create a `BoredPetsNFT.sol` file under `packages/truffle/contracts`. :
 
-## Getting Started
+```
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.22 <0.9.0;
 
-- Do this
-- Do that
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
 
-## Contract
+contract BoredPetsNFT is ERC721URIStorage {
+  using Counters for Counters.Counter;
+  Counters.Counter private _tokenIds;
 
-...
+  constructor() ERC721("Bored Pets Yacht Club", "BPYC") {}
+
+  function mint(string memory _tokenURI) public {
+    _tokenIds.increment();
+    uint256 newTokenId = _tokenIds.current();
+    _safeMint(msg.sender, newTokenId);
+    _setTokenURI(newTokenId, _tokenURI);
+  }
+}
+```
+Let's take a look at the imports:
+
+- `@openzeppelin/contracts/token/ERC721/ERC721.sol`
+
+To be a valid NFT, `BoredPetsNFT` implements the ERC-721 standard by inheriting the implementation of `ERC721URIStorage.sol`
```

Comment:
Do we need the text "To be a valid NFT," Maybe just use:
"The `BoredPetsNFT`"

Only a suggestion, minimize text where we can?